Dockerfile

```diff
@@ -1 +1,11 @@
-# empty
\ No newline at end of file
+from python:3.11.2-slim-buster
```

Comment:
The Dockerfile instructions should be in Uppercase always because Best Practices.